scripts/env/dev/init.sh

```diff
@@ -190,6 +191,9 @@ ansible-playbook -i ./hosts_local --connection local setup_s3dev_centos77_8.yml
 
 rm -f ./hosts_local
 
+# copy s3 support bundle files rotation script for retaining fixed number of latest s3 support bundles
```

Comment:
This can be done as part of ansible script.
e.g. pls check cortx-s3server/ansible/common-runtime.yaml and  search for "s3logfilerollover.sh" in that file.

Reply:
I will check this out.

Reply:
done